hw/top_earlgrey/formal/conn_csvs/top_earlgrey_conn.csv

```diff
@@ -2,6 +2,10 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 #
+# Run these checks with
+#  cd hw/top_earlgrey/formal
+#  ../../../util/dvsim/dvsim.py chip_conn_cfg.hjson --local
```

Comment:
Can we get rid of the --local switch? I add this one because it runs faster on your local machine :)

Reply:
Done